src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionRequestValidator.java

```diff
@@ -17,69 +16,17 @@
 
     /**
      * Validates whether the ActionRequest should be sent to a handler service.
-     * Contains the business logic for deciding whether the recipient is to receive an email or letter,
-     * dependent on the status of their account and the response.
-     * @param actionType
-     * @param actionRequest
-     * @return isValid
+     * Checks if the CaseGroupStatus is completed, this stops any actions beings sent for BI
+     * Cases where other BI's have completed the survey
      */
-    public boolean validate(final ActionType actionType, final ActionRequest actionRequest) {
-        String handler = actionType.getHandler();
-        // Completed no action required
-        if (caseCompleted(actionRequest) || caseCompletedByPhone(actionRequest)) {
-            return false;
-        }
-
-        if (isEmail(handler) && hasActiveRespondent(actionRequest) && enrolmentEnabled(actionRequest)) {
-            return validateEmail(actionType, actionRequest);
-        }
-
-        if (isLetter(handler)) {
-            return validateLetter(actionRequest);
-        }
-
-        log.info("Invalid action request: handler = {} : respondentStatus {} :  enrolmentStatus {} : actionTypePK {}",
-                handler, actionRequest.getCaseGroupStatus(), actionRequest.getEnrolmentStatus(),
-                actionType.getActionTypePK());
-
-        return false;
-    }
-
+    public boolean validate(final ActionRequest actionRequest) {
+        boolean isValid = true;
 
-    private boolean validateEmail(final ActionType actionType, final ActionRequest actionRequest) {
-        if (isNotificationEmail(actionType)) {
-            return true;
-        }
-        if (isReminderEmail(actionType) && (caseInProgress(actionRequest) || caseNotStarted(actionRequest))) {
-            return true;
-        }
-        return false;
-    }
-
-    private boolean validateLetter(final ActionRequest actionRequest) {
-        if (hasNoRespondent(actionRequest) && hasNoEnrolment(actionRequest) && caseNotStarted(actionRequest)) {
-            return true;
-        }
-        if (enrolmentPending(actionRequest) && hasCreatedRespondent(actionRequest) && caseNotStarted(actionRequest)) {
-            return true;
+        if (caseCompleted(actionRequest) || caseCompletedByPhone(actionRequest)) {
+            isValid = false;
```

Comment:
Can these lines be replaced with 
`boolean isValid = !(caseCompleted(actionRequest) || caseCompletedByPhone(actionRequest))`?